salt/redash/files/env

```diff
@@ -0,0 +1,14 @@
+PYTHONUNBUFFERED=0
+REDASH_LOG_LEVEL=INFO
+REDASH_REDIS_URL=redis://redis:6379/0
```

Comment:
Would it be relevant to pull Redis out of Docker, like PostgreSQL? Just wondering. I'm fine to leave it in.